components/org.wso2.carbon.identity.password.expiry/src/main/java/org/wso2/carbon/identity/password/expiry/internal/EnforcePasswordResetComponentDataHolder.java

```diff
@@ -21,6 +21,7 @@
 import org.wso2.carbon.identity.governance.IdentityGovernanceService;
```

Comment:
Let's update the license header for the files which have ben changed.

Reply:
Changed